python/cudf/cudf/core/column/string.py

```diff
@@ -5486,13 +5507,20 @@ def binary_operator(
                         na_rep=cudf.Scalar(None, "str"),
                     ),
                 )
-            elif op in {"eq", "ne", "gt", "lt", "ge", "le", "NULL_EQUALS"}:
+            elif op in {
+                "__eq__",
```

Comment:
I'm going to push this to future work since I will probably define these operators in a single consistent location in a subsequent PR since we create the same set in multiple different files and functions.